students/Matt_Hudgins/Lesson03/slicing_lab.py

```diff
@@ -0,0 +1,91 @@
+'''
+    File Name: slicing_lab.py
+    Author: Matt Hudgins
+    Date created: 4/21/18
+    Date last modified: 4/21/18
+    Python Version 3.6.4
+'''
+# The following sequences will be used for the slicing lab exercise
+data = [3, 1, 4, 1, 5, 9, 2, 6, 5, 3, 5]
+string = 'The first ten digits of pi!'
+
+
+def exchange_first_last(seq):
+    """ This function when called will exchange the first and last
+    items of the sequence"""
+    return(seq[-1:] + seq[1:-1] + seq[0:1])
+
+
+def every_other_removed(seq):
+    """ This function when called will remove every other item in the
+    sequence"""
+    return(seq[::2])
+
+
+def every_other_remove_with_firstlast4_removed_aswell(seq):
+    """ This function when called will remove the first and last items
+    of the sequence and in addition remove every other item in the
+    sequence"""
+    firstlast4removed = seq[4:-4]
+    return(firstlast4removed[::2])
+
+
+def a_quick_switch_aroo(seq):
+    """ This function when called will reverse the items in the
+    sequence"""
+    return(seq[::-1])
+
+
+def another_swith_aroo(seq):
+    """ This function when called will switch the middle third then
+    last third, then the first third in the new order."""
+    first_third = int(len(seq) / 3)
```

Comment:
You can use Python's floor division operator (`//`) to handle sequences that aren't divisible by three